src/components/Todos/Topbar/filter-todo.component.jsx

```diff
@@ -15,31 +19,65 @@ const FilterTodo = () => {
   );
 
   const filterButtons = [
-    { label: LABEL_FILTER_ALL },
-    { label: LABEL_FILTER_COMPLETE },
-    { label: LABEL_FILTER_INCOMPLETE },
+    { value: LABEL_FILTER_ALL, label: LABEL_FILTER_ALL },
+    { value: LABEL_FILTER_COMPLETE, label: LABEL_FILTER_COMPLETE },
+    { value: LABEL_FILTER_INCOMPLETE, label: LABEL_FILTER_INCOMPLETE },
```

Comment:
If both value and label are same, why dont we simplify the array ?
```suggestion
    const filterButtons = [LABEL_FILTER_ALL, LABEL_FILTER_COMPLETE, LABEL_FILTER_INCOMPLETE]
```

Reply:
For React-Select, both value and label properties are mandatory for each option object in the options(filterButtons) array. Without them, the component will not be able to render the options correctly or provide the necessary data when an option is selected.

for a reference: https://react-select.com/home
Thanks!